appveyor.yml

```diff
@@ -8,7 +8,7 @@ environment:
 
 cache:
   - .eslintcache
-  - node_modules
+  - node_modules -> package.json
```

Comment:
This is the syntax Appveyor supports for saying "cache `node_modules` based on the contents of `package.json`" and this should help prevent cases like #6793 which I believe are down to cache invalidation not working as expected.